app/client/src/ce/utils/AnalyticsUtil.tsx

```
@@ -1,375 +1,156 @@
+function reset() {
+  // TODO: Fix this the next time the file is edited
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const windowDoc: any = window;
 
-  static setBlockErrorLogs(value: boolean) {
-    AnalyticsUtil.blockErrorLogs = value;
+  if (windowDoc.Intercom) {
+    windowDoc.Intercom("shutdown");
   }
+
+  segmentAnalytics && segmentAnalytics.reset();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)